apps/desktop/src/mainview/index.css

```
@@ -22,4 +25,258 @@ body {
+  background: transparent;
+  overflow: hidden;
+}
+
+body:has(.clipboard-editor-window) {
+  overflow: hidden;
+}
+
+.clipboard-window {
+  --clipboard-search-placeholder: var(--foreground-placeholder);
+  --clipboard-search-text: var(--foreground);
+  min-height: 100dvh;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)